modules/optimization/test/tests/executioners/constrained/inequality/measurementData.csv

```diff
@@ -0,0 +1,3 @@
+measured_value,coordx,y,z
+207,0.2,0.2,0
```

Comment:
Don't you need all four measurement points?  I'm surprised this works, isn't it underconstrained.  You can skip the csv file and put the measurements directly into the optimizationReporter like I showed above.